truss-chains/truss_chains/stub.py

```diff
@@ -12,6 +13,13 @@
 class BasetenSession:
     """Helper to invoke predict method on Baseten deployments."""
 
+    _client_cycle_time_sec: ClassVar[int] = 3600 * 8  # 8 hours.
```

Comment:
should we just have this be a file-level constant?

Reply:
I'd make it all caps, but I think it makes sense to scope it to the `BasetenSession` session class only, there's no need to expose this outside.